docs/doxygen/config.doxyfile

```diff
@@ -468,7 +475,7 @@ LOOKUP_CACHE_SIZE      = 0
 # DOT_NUM_THREADS setting.
 # Minimum value: 0, maximum value: 32, default value: 1.
 
-# NUM_PROC_THREADS       = 1
+NUM_PROC_THREADS       = 1
```

Comment:
Same as before, is this related to updating the version?

Reply:
The file was updated automatically by `doxygen -u` command. It made apt changes wherever necessary.